src/components/snackbar/snackbar.scss

```diff
@@ -9,44 +9,46 @@
     box-sizing: border-box;
 }
 
-.mdc-snackbar {
-    top: var(--snackbar-top, auto);
-    right: var(--snackbar-right, 0);
-    bottom: var(--snackbar-bottom, 0);
-    left: var(--snackbar-left, 0);
+aside {
+    background: none;
+    border: none;
+    inset: unset;
+    top: 0;
+    right: 0;
+    // margin: unset;
+    // top: var(--snackbar-top, auto);
+    // right: var(--snackbar-right, 0);
+    // bottom: var(--snackbar-bottom, 0);
+    // left: var(--snackbar-left, 0);
 
```

Comment:
we shouldn't remove this feature

Reply:
How should we implement it? What is the need of the consumer?